src/ProgressCircle.webmodeler.ts

```diff
@@ -11,11 +12,22 @@ export class preview extends Component<ContainerProps, {}> {
     }
 
     render() {
-        return createElement(ProgressCircle, this.transformProps(this.props));
+        const { onClickEvent, page, microflow } = this.props;
+        const warnings = ProgressCircleContainer.validateProps(onClickEvent, page, microflow);
```

Comment:
Had a comment from Johan one time...why pass all the props when you just need three??